$NetBSD: patch-meson.build,v 1.1 2018/12/07 20:02:58 schmonz Exp $

Comment out tests (build broken on OS X, not being offered as
TEST_TARGET yet anyway).

--- meson.build.orig	2018-10-22 21:53:04.000000000 +0000
+++ meson.build
@@ -82,6 +82,6 @@ subdir('gsettings')
 subdir('client')
 subdir('bin')
 subdir('docs')
-subdir('tests')
+# subdir('tests')
 
 meson.add_install_script('meson_post_install.py', gio_module_dir)
